robot_sort/robot_sort.py

```diff
@@ -96,10 +72,78 @@ def sort(self):
         """
         Sort the robot's list.
         """
-        # Fill this out
-        pass
+        # Ok so Selection Sort!
+#for i in range(0, len(arr) - 1):
+#   cur_index = i
+#   for sec_index in range(cur_index, len(arr)):
+#       if arr[sec_index] < arr[cur_index]:
+#       arr[sec_index], arr[cur_index] = arr[cur_index], arr[sec_index]
+        global count
```

Comment:
If count is for testing, then that's ok. But if you use count in the process. It's breaking the rules. Check the rules in the README.md

Reply:
the readme wasnt in the same folder as the robot so i missed it until QA ( >.<; ), but the second version is free of extra variables!